mysql_scripts/table_creation/FullDB.sql

```diff
@@ -73,9 +73,16 @@ CREATE TABLE Client (
 
 CREATE TABLE Appointment (
 	appointmentId INTEGER UNSIGNED PRIMARY KEY NOT NULL AUTO_INCREMENT,
-	scheduledTime DATETIME NOT NULL,
+    scheduledTime DATETIME NOT NULL,
 	createdAt DATETIME NOT NULL DEFAULT NOW(),
-	arrivedAt DATETIME NULL,
+    language VARCHAR(255) NOT NULL,
+	timeIn DATETIME NULL DEFAULT NULL,
+    timeReturnedPapers DATETIME NULL DEFAULT NULL,
+    actualAppointmentTime DATETIME NULL DEFAULT NULL,
+    timeFinished DATETIME NULL DEFAULT NULL,
```

Comment:
Just to verify, these column names are the ones that Vicky said they would want?

Reply:
Yeah, I looked back at my notes and this is what she had said she wanted. Essentially I just used the existing code with buttons instead of lanes.

Reply:
I think that the `actualAppointmentTime` column should be renamed to `timeAppointmentStarted`. And perhaps rename `timeFinished` to `timeAppointmentEnded` so that we can be consistent between the start/end times? Also, I say ended because, to me, the word "finished" implies success, but ended doesn't necessarily imply success. It really doesn't matter though, but if you think that argument makes sense, feel free to change it. 

One other super important thing. How does one reconnect an appointment to what volunteer did that appointment? Your current structure doesn't allow that, right? At present, there is a table in the database called `ServicedAppointment` that was supposed to represent an appointment that a volunteer had at least started (not necessarily finished). I think you could use that table for these datetime columns you've added. And I think we can use that `ServicedAppointment` table to hold all this metadata-type information that goes along with an appointment. Additionally, that table already has a column for connecting it to the volunteer who did it (the userId column). You should also probably make that userId column nullable. We may want to rename the `ServicedAppointment` table, and feel free to if you think of a better name. 

Connecting who did an appointment with the appointment is extremely important for reporting purposes, and for correction purposes (if someone was doing something wrong, we have to know who all it effected). 

Again, consult https://github.com/UNL-CSE-AMBASSADORS/VITA/pull/96 to see the ERD and see what I mean.